examples/vacationsApprovalServerless/README.md

```diff
@@ -0,0 +1,89 @@
+# Vacation Approval App example using Serverless Framework
+
+With Smartmate you have the ability to call custom endpoints in order to have manage custom logic outside the Smartmate environment.
+
+This example explains how to setup a Smartmate App using Serverless Framework.
+
+
+`Note`: With Serveless Framework you can use many Cloud Providers, in this case we will be using AWS.
+## Setup Serveless
+
+In order to setup a Serverless funtion, first you need to install the Serverless Framework:
+```
+   npm install -g serverless
```

Comment:
this command has spaces before the command, and the next one doesn't. Standardize, so it looks similar.